hub/alembic/versions/26e1e353eb58_create_run_scheduler_table.py

```diff
@@ -0,0 +1,37 @@
+"""create run_scheduler table.
+
+Revision ID: 26e1e353eb58
+Revises: c5af2948a246
+Create Date: 2025-01-13 14:57:15.478166
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "26e1e353eb58"
+down_revision: Union[str, None] = "c5af2948a246"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "schedule_runs",
```

Comment:
I think `scheduled_runs` would be clearer. Once the come into this table they have been scheduled.
```suggestion
        "scheduled_runs",
```